pkg/cache/informers.go

```diff
@@ -59,16 +58,6 @@ func initCacheInformers(instance *Store, config *rest.Config, stopCh <-chan stru
 	crdFactory := crdinformers.NewSharedInformerFactoryWithOptions(crdClientSet, 0)
 
 	podInformer := factory.Core().V1().Pods().Informer()
-	modelInformer := crdFactory.Model().V1alpha1().ModelAdapters().Informer()
-
-	defer runtime.HandleCrash()
```

Comment:
HandleCrash is usually used in goroutinue or in functions that contain functions that are not sure whether a panic will occur or not.
it doesn't seem necessary in here.